packages/mgt-components/src/graph/graph.user.ts

```diff
@@ -190,22 +191,30 @@ export async function getUsersForUserIds(
 
   for (const id of userIds) {
     peopleDict[id] = null;
+    let apiUrl: string = `/users/${id}`;
     let user = null;
     if (getIsUsersCacheEnabled()) {
       user = await cache.getValue(id);
     }
-    if (user && getUserInvalidationTime() > Date.now() - user.timeCached) {
+    if (user && user?.user && getUserInvalidationTime() > Date.now() - user.timeCached) {
```

Comment:
We don't need the first part here as it's implicitly included in the chained optional access.
```suggestion
    if (user?.user && getUserInvalidationTime() > Date.now() - user.timeCached) {
```